modelcluster/forms.py

```diff
@@ -177,12 +177,15 @@ def __new__(cls, name, bases, attrs):
                 except AttributeError:  # thrown if opts.widgets is None
                     widgets = None
 
-                kwargs = {
-                    'extra': cls.extra_form_count,
-                    'formfield_callback': formfield_callback,
-                    'fk_name': rel.field.name,
-                    'widgets': widgets
-                }
+                try:
+                    kwargs = {
+                        'extra': cls.extra_form_count,
+                        'formfield_callback': formfield_callback,
+                        'fk_name': rel.field.name,
+                        'widgets': widgets
+                    }
+                except AttributeError:
+                    continue
```

Comment:
Same issue as https://github.com/torchbox/django-modelcluster/pull/36/files#r73386914. If we need special-case handling for M2M fields here, there should be a test for that. (As it is, if I remove this try/except, the only test failures are ones relating to `ClusterTaggableManager`.)